examples/all-clusters-minimal-app/nrfconnect/README.md

```diff
@@ -44,6 +40,13 @@ device works as a Thread Minimal End Device.
 
 <hr>
 
+## IPv6 network support
+
+The development kits for this sample offer the following IPv6 network support for Matter:
+
+* Matter over Thread is supported for ``nrf52840dk_nrf52840``, ``nrf5340dk_nrf5340_cpuapp``, and ``nrf21540dk_nrf52840``.
+* Matter over Wi-Fi is supported for ``nrf5340dk_nrf5340_cpuapp`` with the ``nrf7002_ek`` shield attached or for ``nrf7002dk_nrf5340_cpuapp``.
```

Comment:
Move this section under Requirements, on the same level as Supported devices (###). Also update the table of contents.